src/actions/Button.scss

```diff
@@ -0,0 +1,181 @@
+.button {
+  --border-width: var(--bloom-border-2);
+  --font-family: var(--bloom-font-alt-sans);
+  --font-weight: bold;
+  --interior-gap: var(--bloom-s3);
+  --icon-size-percentage: 75%;
+  --icon-side-padding: var(--bloom-s4);
+
+  --padding-sm: calc(var(--bloom-s3_5) - var(--border-width))
+    calc(var(--bloom-s5) - var(--border-width));
+  --font-size-sm: var(--bloom-font-size-sm);
+  --border-radius-sm: var(--bloom-rounded);
+  --padding-md: calc(var(--bloom-s4) - var(--border-width))
+    calc(var(--bloom-s6) - var(--border-width));
+  --font-size-md: var(--bloom-font-size-base);
+  --border-radius-md: var(--bloom-rounded);
+  --padding-lg: calc(var(--bloom-s5) - var(--border-width))
+    calc(var(--bloom-s7) - var(--border-width));
+  --font-size-lg: var(--bloom-font-size-lg);
+  --border-radius-lg: var(--bloom-rounded);
+
+  all: initial;
+  display: inline-flex;
+  gap: var(--interior-gap);
+  padding: var(--padding-md);
+  border-style: solid;
+  border-width: var(--border-width);
+  border-radius: var(--border-radius-md);
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  font-size: var(--font-size-md);
+  line-height: var(--bloom-line-height-tight);
+  cursor: pointer;
+
+  &[data-size="sm"] {
+    padding: var(--padding-sm);
+    font-size: var(--font-size-sm);
+    border-radius: var(--border-radius-sm);
+  }
+
+  &[data-size="lg"] {
+    padding: var(--padding-lg);
+    font-size: var(--font-size-lg);
+    border-radius: var(--border-radius-lg);
+  }
+
+  &[data-variant="primary"] {
+    background-color: var(--bloom-color-primary);
+    border-color: var(--bloom-color-primary);
+    color: var(--bloom-color-on-inverse);
+
+    &:hover {
+      background-color: var(--bloom-color-primary-dark);
+      border-color: var(--bloom-color-primary-dark);
+    }
+  }
+
+  &[data-variant="primary-outlined"] {
+    background-color: var(--bloom-bg-color-surface);
+    border-color: var(--bloom-color-primary);
+    color: var(--bloom-color-primary);
+
+    &:hover {
+      border-color: var(--bloom-color-primary-dark);
+      color: var(--bloom-color-primary-dark);
```

Comment:
It might just be my monitor, but it is really hard to notice the hover state for primary outlined. The darker blue is a bit too subtle of a difference on a small border. I'm curious if there is a recommended contrast for accessibility on the hover state?

Reply:
Yeah, I would agree it's pretty subtle. We should check in with @slowbot

Reply:
@slowbot Ah, that makes sense, it's like how the previous button hover worked. I'll move forward with that update.